core/src/main/java/nl/nn/adapterframework/pipes/XmlSwitch.java

```diff
@@ -244,7 +244,7 @@ public void setEmptyForwardName(String empty){
 	}
 
 	/**
-	 * If set to <code>2</code> xslt processor 2.0 (net.sf.saxon) will be used, otherwise xslt processor 1.0 (org.apache.xalan). <code>0</code> will auto detect
+	 * If set to <code>2</code> xslt processor 2.0 (net.sf.saxon) will be used, otherwise xslt processor 1.0 (org.apache.xalan). <code>0</code> will auto detect. This pipe supports up to and including XSLT version 3.0
```

Comment:
Actually this comment should be updated to say that the Saxon processor will be used when setting this parameter to 2 or 3.